packages/lodestar-validator/src/services/attestation.ts

```diff
@@ -170,7 +171,9 @@ export class AttestationService {
 
     if (duty.isAggregator) {
       setTimeout(() => {
-        this.aggregateAttestations(duty.attesterIndex, duty, attestation, fork, genesisValidatorsRoot);
+        if (attestation) {
```

Comment:
we Already check that 6 lines above

Reply:
I know but since the variable is declared with `let` Typescript thinks that the variable can change to undefined again between the check and the execution of the timeout. So it considers that attestation may be undefined within the timeout block